services/rabbit/Makefile

```diff
@@ -0,0 +1,66 @@
+REPO_BASE_DIR := $(shell git rev-parse --show-toplevel)
+
+include ${REPO_BASE_DIR}/scripts/common-services.Makefile
+# common-services.Makefile should be included first as common.Makefile
+# relies on STACK_NAME var which is defined in common-services.Makefile
+include ${REPO_BASE_DIR}/scripts/common.Makefile
+
+#
+# Operations
+#
+
+include ${REPO_BASE_DIR}/services/rabbit/.operations.Makefile
+
+#
+# Docker compose files
+#
+
+### Load Balancer
+docker-compose.loadbalancer.yml: docker-compose.loadbalancer.yml.j2 \
+					.env \
+					configs/rabbitmq.conf \
+					configs/erlang.cookie.secret \
+					configs/haproxy.cfg \
+					venv \
+					$(VENV_BIN)/j2
+	@$(call jinja, $<, .env, $@)
+
+.stack.loadbalancer.yml: docker-compose.loadbalancer.yml .env
+	@${REPO_BASE_DIR}/scripts/docker-stack-config.bash -e .env $< > $@
+
+### Node
+
+.PRECIOUS: node0%.env
+node0%.env: .env
+	envsubst < $< > $@; \
+	echo NODE_INDEX=$* >> $@
+
+.PRECIOUS: docker-compose.node0%.yml
+docker-compose.node0%.yml: docker-compose.node0x.yml.j2 \
```

Comment:
cool stuff with the `%`, a bit hard to read if one doesnt know makefiles but we are makefile experts :D